CefSharp.Core/Internals/TypeConversion.h

```diff
@@ -149,6 +149,18 @@ namespace CefSharp
                 {
                     cefValue->SetDouble(Convert::ToDouble(value));
                 }
+				else if (type == List<Object^>::typeid)
```

Comment:
Sorry, I thought I had the space settings for all languages. I've pushed the changes.